### ISSUE TITLE
Frame naming

### ISSUE THREAD

**Issue by Talon1024** [nashmuhandes/io_export_gzdoom_md3#9]
MD3 supports named frames, so what is the best way to go about choosing names for the individual frames?

**Comment by nashmuhandes** [comment]
Default: `framexxx` (xxx is frame number)

Alternatively - you can drop markers in the timeline view (by pressing M) - use marker names for the prefix?

**Comment by nashmuhandes** [comment]
If you do this, can you backport it to 2.79 too please 🙏

**Comment by Talon1024** [comment]
Not to worry. I plan to backport every feature I add.

Also, I'm not familiar with Blender's animation system... Can an object have multiple actions? What is an "action" anyway?

**Comment by nashmuhandes** [comment]
Yes, an armature can have multiple actions associated with it. Actions are basically a way for you to organize and separate different animations.

For example:

We are traditionally used to placing ALL animations (like standing still, walking, running, jumping, attacking, death, etc) in a single timeline. We may then choose to organize the timeline by placing markers on the timeline, so we know where each animation starts and ends.

ALTERNATIVELY - we can create a separate action for each animation. Standing would go on its own Action, walking on another, etc.

As for how this translates to game assets - this is engine-dependent. Since we are talking about GZDoom, and MD3 - this isn't really applicable to us. MD3 only supports a single timeline. So I'm not sure if trying to incorporate the Action mindset/workflow would be relevant or even possible here.

**Comment by Talon1024** [comment]
I would use the name of the action to name the frames. However, I guess I could use the timeline marker-based naming if there's only one action on the timeline, and then if there are multiple actions overlapping each other on the timeline, I could put them into alphabetical order (or something), and then concatenate them onto a single timeline.
